File: Domain/Activity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class Activity // När vi skapar en new Activity retunerar vi ett objekt med dessa properties
    {
        public Guid Id { get; set; } // Guid kommer från System. "Id" är även primery key.
        public string Title { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }
        public string City { get; set; }
        public string Venue { get; set; }
    }
}

File: client-app/src/app/models/activity.ts
export interface Activity {
  id: string;
  title: string;
  date: string; // Date kommer användas senare
  description: string;
  category: string;
  city: string;
  venue: string;
}

export enum ActivityPropNames {
  id = 'id',
  title = 'title',
  date = 'date',
  description = 'description',
  category = 'category',
  city = 'city',
  venue = 'venue'
}